modules/new_user/new_user.py

```diff
@@ -4,16 +4,16 @@
 
 def is_unassigned(member):
 	#if the member has the unassigned role
-	return (get(member.roles, id=int(get_id('UNASSIGNED_ROLE_ID'))) != None)
+	return (get(member.roles, id=get_id('UNASSIGNED_ROLE_ID')) != None)
 
 
 async def add_role(ctx, machon, year):
 	'''adds the right role to the user that used the command'''
 	member = ctx.author
 
 	#formatting role for env file
-	role_id = machon.upper() + '_YEAR_' + year + "_ROLE_ID" #format example: LEV_YEAR_1_ROLE_ID
-	new_role = get(member.guild.roles, id = int(get_id(role_id)))
+	role_id = machon.strip(',').upper() + '_YEAR_' + year + "_ROLE_ID" #format example: LEV_YEAR_1_ROLE_ID
```

Comment:
The whole point of using the "," was to split the user input on it. For example, suppose someone has two first names and they always go by the two names. The⊥wouldn't accept that if it does the during on a space (which is what's done by using arguments). What your code should do is take the entire command (minus ~join) and split that on a "," (then strip each element to remove extra spaces). That way it uses the "," to separate arguments